File: emanation/pingsweep.py
#!/usr/bin/python

import subprocess
import os
devnull = open(os.devnull, 'w')

for ip in range(1,255):
    nid="10.10.48.{0}".format(ip)
    run=subprocess.call(['ping', '-c', '1', '-w 1', nid],stdout=devnull)
    if run==0:
        print nid

File: emanation/pingsweep.rb
#!/usr/bin/ruby
range = 1..254

for ip in range 
    ip_address = "192.168.99.#{ip}"
    puts `ping -c1 #{ip_address} |grep "bytes from" |cut -d " " -f4 | cut -d":" -f1 &`
end

